### DISCUSSION TITLE
[4.x]: Support order duplication from the element index actions.

### DISCUSSION THREAD

**Discussion by lukeholder** [craftcms/commerce#3593] [Ideas]
### Description

The base element added a 'duplicate' action, which Orders do not yet support.

### Craft CMS version

4.x

### Craft Commerce version

4.x

### Installed plugins and versions

-

**Comment by jelmer-scoremedia** [comment]
Any ETA on when this will be resolved?

**Comment by jonleverrier** [comment]
Has this been shipped? The button appears in the control panel on CraftCMS/Commerce v4?
